sources/packages/web/src/components/common/modals/ConfirmExtendTime.vue

```diff
@@ -43,65 +41,30 @@ export default {
       required: true,
       default: "" as ClientIdType,
     },
+    countdown: {
+      type: Number,
+      required: true,
+    },
+    showDialog: {
```

Comment:
bit confused.  What is happing here? is it overriding `showDialog` from `useModalDialog<boolean>();`

Reply:
ok. I changed it to use `showDialog`. Thanks!